src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineIsolationTest.java

```diff
@@ -77,13 +101,189 @@ public void readUncommittedTest() throws DatabaseFactoryException {
 
     @Test
     public void repeatableReadTest() throws DatabaseFactoryException {
-        this.properties.setProperty(ISOLATION_LEVEL, "read_uncommitted");
+        this.properties.setProperty(ISOLATION_LEVEL, "repeatable_read");
         DatabaseFactory.getConnection(properties);
     }
 
     @Test
     public void serializableTest() throws DatabaseFactoryException {
-        this.properties.setProperty(ISOLATION_LEVEL, "read_uncommitted");
+        this.properties.setProperty(ISOLATION_LEVEL, "serializable");
```

Comment:
these were just plain wrong? they are unrelated to the change, right?

Reply:
Yes, I noticed this because of the new SQL server "transaction snapshot" isolation level.
They were wrong in the sense that the isolation level simply didn't match what the test name implies - the tests were working but testing the wrong values :smile: (since ever)